test/util/compare-result.js

```diff
@@ -0,0 +1,18 @@
+const { expect } = require("chai");
+const jscodeshift = require("jscodeshift");
+const { compile } = require("coffee-script");
+
+module.exports = (testName, transform, source) => {
+  it(testName, () => {
+    const ES5Source = compile(source, { bare: true });
+    const transformedSource = transform({ ES5Source }, { jscodeshift });
+
+    // We expect the transform to change the output
+    expect(transformedSource).not.to.be.null;
+
+    /* eslint-disable no-eval */
```

Comment:
If we decide to go through with this we could probably best add this rule to the `.eslintrc` in the tests folder